modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -9,15 +9,11 @@
 [id="olm-restricted-networks-operatorhub_{context}"]
 = Disabling the default OperatorHub sources
 
-Operator catalogs that source content from Quay.io are configured for
-OperatorHub by default during an {product-title} installation. Before
-configuring OperatorHub to instead use local catalog sources in a restricted
-network environment, you must disable the default catalogs.
+Operator catalogs that source content provided by Red Hat and community projects are configured for OperatorHub by default during an {product-title} installation. Before configuring OperatorHub to instead use local catalog sources in a restricted network environment, you must disable the default catalogs.
```

Comment:
Is there a particular reason why this is just for restricted networks? I keep hearing that some customers want to use their own registries even if they're not in restricted networks. This is what's in the assembly about mirroring the installation content:
> You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.

Unless your stakeholders are telling you otherwise, you might want to discuss adding something similar to the custom catalog topics.

Reply:
I left it out of "Managing custom catalogs" initially because, while it was required for the restricted networks procedure, it wasn't critical to just general custom catalog usage. However, you're right, I should include it in the custom catalog assembly just as an option, at a level higher and not in the main-line path of the other procedures. I'll do so in a follow-up PR.